lib/internal/bootstrap/pre_execution.js

```diff
@@ -88,6 +88,13 @@ function initializeDeprecations() {
                 'DEP0103') :
       types[name];
   }
+
+  // TODO(joyeecheung): this is a legacy property exposed to process.
+  // Now that we use the config binding to carry this information, remove
+  // it from the process. We may consider exposing it properly in
+  // process.features.
+  const { noBrowserGlobals } = internalBinding('config');
+  process._noBrowserGlobals = noBrowserGlobals;
```

Comment:
This changes the behavior to the former.

Before, this variable was only set in case it was true. Now it's set unconditionally.
It is now also set as a regular property and not only as read-only.